src/main/java/seedu/expense/logic/commands/TopupCommand.java

```diff
@@ -27,6 +28,7 @@
             + PREFIX_TAG + "Food";
 
     public static final String MESSAGE_SUCCESS = "New budget amount for %s: $%.02f";
+    public static final String MESSAGE_CATEGORY_NOT_FOUND = "The expense book does not contain the category %s";
```

Comment:
I think the message could be more detailed: "The expense book does not contain the category %s. Please add it using the addCat command first."

Reply:
However, can we look into using `CategoryNotFoundException `for consistency?